controllers/appointmentsController.ts

```diff
@@ -21,22 +21,34 @@ const create = async (req: Request, res: Response): Promise<void> => {
   // Todo : test case where patient id does not exist
   validation.checkErrors(req);
 
+  const beginningDate = new Date('2021-03-21');
   const date = new Date(req.body.date);
+  const today = new Date();
+  const diffInMonth = Math.abs(date.getMonth() - today.getMonth());
   const { patientId } = req.body;
   const psyId = req.user.psychologist;
   const patientExist = await dbPatient.getById(patientId, psyId);
 
   if (patientExist) {
-    await dbAppointments.insert(date, patientId, psyId);
-    console.log(
-      `Appointment created for patient id ${patientId} by psy id ${psyId}`,
-    );
-    res.json({
-      message: `La séance du ${dateUtils.formatFrenchDate(date)} a bien été créée.`,
-    });
+    if (date > today && diffInMonth > 4) {
```

Comment:
```suggestion
    if (diffInMonth > 4) {
```
today is useless

Reply:
Alors du coup y a un petit souci.
Enfaite si la date est par exemple après le 21 mars 2021 ( disant le 23 mars 2021 ).
Ce devrait être bon, mais le fait que la condition des 4 mois est la bah ca ne crée pas l'appointment vu que ca dépasse des 4 mois.